phpunit.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
[Coveralls](https://coveralls.io) is one option, but I have had difficulty implementing a nice solution for Drupal modules via TravisCI without running code coverage for every PHP version. It also puts a composer dependency into the module on php-coveralls.

What I am currently doing is running phpunit with --coverage-text, and printing out that report in TravisCI, but that will not directly report back to GitHub what the new coverage is. TravisCI will allow to run the tests automatically for PHP Unit tests.

A sample .travis.yml file:
```yaml
language: php

php:
  - 5.5
  - 5.6

sudo: false

install:
  # Download Drupal.
  - TESTDIR=$(pwd)
  - cd ..
  - git clone --branch 8.0.x --depth 1 http://git.drupal.org/project/drupal.git drupal
  - cd drupal

before_script:
  # rsync the module directory into modules/
  - rsync -rtlDPvc --exclude .git/ $TESTDIR modules/
  - cd modules/key
script:
  - ../../core/vendor/bin/phpunit --coverage-text
```